docs/core/project-sdk/msbuild-props.md

```diff
@@ -0,0 +1,201 @@
+---
+title: MSBuild properties for Microsoft.NET.Sdk
```

Comment:
Instead of having a page for each SDK, shouldn't this simply be the reference? We can still link to the other specific ones in ASP.NET from the TOC but my understanding is that a lot of these properties would be common across SDKs no? Like TargetFramework.

Reply:
TargetFramework is common across SDKs because most of the other SDKs reference Microsoft.NET.Sdk. That's not necessarily the case but is for most project SDK types.

Reply:
I do mention that [here](https://review.docs.microsoft.com/en-us/dotnet/core/project-sdk/overview?branch=pr-en-us-16877#available-sdks). Instead of "depend on", should I use the term "reference" instead? Or explicitly state that the other SDKs inherit all the properties from the base SDK?